pytorch_lightning/loops/fit_loop.py

```diff
@@ -225,15 +225,6 @@ def on_run_end(self) -> None:
         # hook
         self.trainer.call_hook("on_train_end")
 
-        # todo: TPU 8 cores hangs in flush with TensorBoard. Might do for all loggers.
-        # It might be related to xla tensors blocked when moving the cpu
-        # kill loggers
-        if self.trainer.logger is not None:
-            self.trainer.logger.finalize("success")
-
-        # summarize profile results
-        self.trainer.profiler.describe()
-
```

Comment:
these need to be added to the trainer's teardown hook

Reply:
My bad! I was merging the commits and somehow missed these changes. Thank you for spotting it!